### ISSUE TITLE
Not able to process credit cards paymets in SAB

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#344]
In GitLab by @kdjstudios on May 16, 2017, 09:30

Amecia Wrote: http://www.servicedesk.answernet.com/profiles/ticket/2017-05-16-75869/conversation

We are not able to process any credit card payments in SAB.
 
When clicking pay by credit card we receive  message saying something went wrong.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 17, 2017, 01:00

@kdjstudios 

Need a little more info than that..
* Account
* Site
* CC payment or payment by clicking on "Accept new Payment button"

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 17, 2017, 08:41

@smasih 

I have requested the additional information from Amecia. We will follow up once received.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 17, 2017, 09:19

@smasih 

Amecia Wrote:

This is for the Chattanooga Site (035). The account that I was trying to do the payment on was Lady Genius acct# 6930. I have already processed the credit card on the back end and I was able to process a credit card today on SAB so it looks like the issue has been resolved. If I come across another issue I will reach back out to you.
